src/lib/utilities/handle-error.ts

```diff
@@ -58,10 +57,25 @@ export const handleUnauthorizedOrForbiddenError = (
   }
 };
 
-export const isUnauthorized = (error: any): boolean => {
-  return error?.statusCode === 401 || error?.status === 401;
+export const isUnauthorized = (error: unknown): error is TemporalAPIError => {
+  return hasStatusCode(error, 401);
+};
+
+export const isForbidden = (error: unknown): error is TemporalAPIError => {
+  return hasStatusCode(error, 403);
 };
 
-export const isForbidden = (error: any): boolean => {
-  return error?.statusCode === 403 || error?.status === 403;
+const hasStatusCode = (
+  error: unknown,
+  statusCode: number | string,
+): error is TemporalAPIError => {
+  if (has(error, 'statusCode')) {
+    return error?.statusCode === statusCode;
```

Comment:
Slight nit: this (and the one below in L77) shouldn't be needed if we're already checking if 'statusCode' is in error, right?